templates/node/events/package-lock.json

```diff
@@ -1,8 +1,2837 @@
 {
   "name": "event-handler",
   "version": "0.1.0",
-  "lockfileVersion": 1,
```

Comment:
Is this file supposed to be part of this renaming PR ?

Reply:
hm, don't know how does this end up here.

Reply:
these files get modified after running make test. The deps needed to be updated.